services/src/contexts/migrations/migration_0016_merge_providers.sql

```diff
@@ -58,3 +58,12 @@ WHERE pro.type_name = 'CopernicusDataspaceDataProviderDefinition';
 
 DROP TABLE pro_layer_providers;
 DROP TYPE "ProDataProviderDefinition";
+
+-- user_sessions
+
+ALTER TABLE sessions ADD COLUMN
+user_id uuid REFERENCES users (id) ON DELETE CASCADE NOT NULL;
```

Comment:
this will fail if  the sessions table is not empty right? I think we'd either need to clear it or migrate the data in user_sessions into the sessions table.

Reply:
[4468c71](https://github.com/geo-engine/geoengine/pull/1009/commits/4468c71d093fb8101500a06653a480858527e9c7)